Moesif.Middleware/Moesif.Middleware.nuspec

```diff
@@ -20,16 +20,16 @@
 		<group targetFramework="netstandard2.0">
 			<dependency id="Microsoft.AspNetCore.Http" version="2.1.0" />
 			<dependency id="Microsoft.AspNetCore.Http.Extensions" version="2.1.0" />
-			<dependency id="Moesif.Api" version="2.0.5" />
+			<dependency id="Moesif.Api" version="2.0.7" />
 			<dependency id="System.ValueTuple" version="4.5.0" />
-			<dependency id="Newtonsoft.Json" version="9.0.1" />
+			<dependency id="Newtonsoft.Json" version="13.0.2" />
             <dependency id="System.IdentityModel.Tokens.Jwt" version="6.8.0" />
 		</group>
 		<group targetFramework="net45">
 			<dependency id="Microsoft.Owin" version="3.1.0" />
```

Comment:
Should we update the version to 4.2.2 too?

Reply:
Done